pkg/config/config.go

```diff
@@ -152,6 +157,15 @@ func NewNSXOperatorConfigFromFile() (*NSXOperatorConfig, error) {
 	return nsxOperatorConfig, nil
 }
 
+func NewNSXOperatorConfigFromFile() (*NSXOperatorConfig, error) {
```

Comment:
It's good to add a UT case to test NewNSXOperatorConfigFromFile returning an error

Reply:
done